pkg/flexvolume/flexvolume.go

```diff
@@ -82,26 +86,6 @@ type Driver interface {
 	Unmount(mountDir string) DriverStatus
 }
 
-// ExitWithResult outputs the given Result and exits with the appropriate exit
-// code.
-func ExitWithResult(result DriverStatus) {
```

Comment:
Whilst hoisting `ExitWithResult` out of `ExecDriver` is a good idea, I don't see where you've re-implemented outputting the JSON to stdout. I'd suggest keeping this code and calling it from `main.go`.